openqa-trigger-bisect-jobs

```diff
@@ -292,6 +311,7 @@ def main(args):
         for job_id in sorted(created_job_ids):
             log.info(f"Created {job_id}")
             created += f"* **{test_name}**: {base_url}/t{job_id}\n"
+            openqa_set_job_prio(job_id, args.url, prio, args.dry_run)
```

Comment:
really this isn't needed

Reply:
Is it really possible to set the priority with `openqa-clone-job`? It doesn't seem to be possible according to `--help`.